extras/docker/Dockerfile

```diff
@@ -1,92 +1,21 @@
-# w3af.org
-# https://github.com/andresriancho/w3af/tree/master/extras
+FROM ubuntu:14.04
 
-FROM ubuntu:12.04
-MAINTAINER Andres Riancho <andres.riancho@gmail.com>
+ADD . /w3af
```

Comment:
This is a bad idea. If you `ADD` a directory to the docker image, the cache will break and all the commands after it will be re-run instead of restored from cache. `ADD` commands should be run as late as possible to speedup builds.